src/container/ImportCAT/ImportCAT.tsx

```diff
@@ -80,54 +86,97 @@ function ImportCAT() {
-                                            iconUrl: item.icon_url,
-                                        }}
-                                        active={selected.some(
-                                            (e) => e.asset_id === item.asset_id
-                                        )}
-                                        disabled={existedAssetIds?.some(
-                                            (id) => id === item.asset_id
-                                        )}
-                                        onClick={() => {
-                                            if (
-                                                selected.some(
-                                                    (e) =>
-                                                        e.asset_id ===
-                                                        item.asset_id
-                                                )
-                                            ) {
-                                                setSelected([
-                                                    ...selected.filter(
+                            <div className="relative top-0 pt-4 pb-2 ml-3 text-body3 text-primary-100">
+                                {selected.length ? (
+                                    <>
+                                        <div className="flex flex-row items-center gap-[9.35px]">
+                                            <span>
+                                                {t('count-selected_token', {
+                                                    count: selected.length,
+                                                })}
+                                            </span>
+                                            <BottomIcon
+                                                onClick={() => {
+                                                    setToggleSelectedList(
+                                                        !toggleSelectedList
+                                                    )
+                                                }}
+                                                className={`w-5 h-5  text-active cursor-pointer transform transition-all ease-in-out ${classNames(
+                                                    `${
+                                                        toggleSelectedList &&
+                                                        '-rotate-180 '
+                                                    }`
+                                                )}`}
+                                            />
+                                        </div>
+                                    </>
+                                ) : (
+                                    <span>
+                                        {t('import_token-search_results')}
+                                    </span>
+                                )}
+                            </div>
```

Comment:
fix logic error when selected token is 0
![import cat](https://user-images.githubusercontent.com/13610603/227822207-849fb100-7f92-439c-a2da-35ebb48a2a74.gif)